dir.targets

```diff
@@ -19,12 +19,17 @@
 
   <Target Name="GetLatestCommitHash">
```

Comment:
I guess I would have put the `Condition=" '$(LatestCommit)' == '' "` on the whole target.  No need to run the target at all if it is already set.

Reply:
Should I do the same to the `GetCommitCount` target below?